examples/accept/README.md

```diff
@@ -14,17 +14,17 @@ import React from 'react';
 import {useDropzone} from 'react-dropzone';
 
 function Accept(props) {
-  const {acceptedFiles, rejectedFiles, getRootProps, getInputProps} = useDropzone({
+  const {acceptedFiles, fileRejections, getRootProps, getInputProps} = useDropzone({
     accept: 'image/jpeg, image/png'
   });
-  
+
   const acceptedFilesItems = acceptedFiles.map(file => (
     <li key={file.path}>
       {file.path} - {file.size} bytes
     </li>
   ));
 
-  const rejectedFilesItems = rejectedFiles.map(file => (
+  const fileRejectionsItems = fileRejections.map(file => (
```

Comment:
export interface FileRejection {
  file: File;
  errors: FileError[];
}

if you want to access files in fileRejections, you'll have to address file property in each fileRejection element